contrib/main.yml.sample

```diff
@@ -229,6 +229,15 @@ datadog_enabled: false
 # Only used if datadog_enabled is true.
 #datadog_api_key:
 
+# signalfx_enabled controls whether or not the SignalFX agent should be
+# deployed across the cluster. If true, signalfx_api_key must be set.
```

Comment:
I don't fully understand it myself, but per the README:

> This tool will auto discover kubernetes cluster nodes and send container and machine metrics to SignalFx. You can deploy it to run in a single pod within a Kubernetes cluster where it will collect data from the cadvisor container resource usage and performance analysis agent (that is integrated into the kubelet binary) using cluster member permissions only.